### DISCUSSION TITLE
Backup current books read information

### DISCUSSION THREAD

**Discussion by emjayit** [PaulWoitaschek/Voice#1668] [Ideas]
Provide a way to backup/export/import the Voice current book status, books read, booksmarks and other details to be able to restore to another device.

**Comment by l0udsil3nc3** [comment]
Very important for because I keep switching or resetting my phone all the time!

**Reply by PaulWoitaschek** [reply]
Here is a tutorial on how to Backup on Android:
https://support.google.com/android/answer/2819582

Does that solve your problem?

**Reply by polyzen** [reply]
I personally would rather start fresh than backup to Google. Backup/Restore support would be appreciated.

**Reply by snonux** [reply]
Yeah, would be great to be able to backup and restore without Google. I, for example, am using a de-googled Android phone and I am syncing all my stuff to NextCloud. The feature must not be NextCloud specific, I would also be happy with a manual export/import functionality!

**Comment by Archie-2021** [comment]
+1 for this one. I recently found out that this project is FOSS , I always loved it program but knowing that just increase my interest much.

I think doing some sort of export is a nice Idea, I know it is a minimalist program and that is what I like about it but notes created there should be moved somewhere eventually. I suggest using markdown as the way of exporting notes from it. It is the best format for note taking (altho plain text will do too). Also using a separate file for each book. We choose a folder for this exported notes and just sync that folder to our other devices via something like syncthing or whatever people prefer to use.

Thanks

**Comment by jaredly** [comment]
fwiw I've got a fix for this ready https://github.com/PaulWoitaschek/Voice/pull/2302 :)
